entries/o/ox.ac.uk.json

```
@@ -0,0 +1,23 @@
+{
+  "University of Oxford": {
+    "domain": "ox.ac.uk",
+    "tfa": [
+      "totp",
+      "sms",
+      "custom-software",
+      "u2f",
+      "call"
+    ],
+    "custom-software": [
+      "Microsoft Authenticator",
+      "Authy"
```

Like in #6892, only the TOTP part of Authy is used.
For a custom-software to be listed it needs to use something other than just RFC-6238 TOTP.
```suggestion
      "Microsoft Authenticator"
```